### DISCUSSION TITLE
How do I shift the favicons of websites in tree style tab to the right?

### DISCUSSION THREAD

**Discussion by ghost** [piroor/treestyletab#2993] [Q&A]
How do I shift the favicons of websites to the right beside the close tab button like this: https://github.com/astroryan12/VerticalTabs/blob/main/imgs/layout-hover1.png .Thanks in advance.

**Comment by piroor** [comment]
It is due to a mis-detection of the sidebar position. It is the appearance for the sidebar placed at right side. When the screen is 1) hi-res (not dot-by-dot) and 2) placed lefter than the primary screen, the calculation logic fails to detect the sidebar position. I researched that at #2984. The auto detection will be disabled in the next release for such cases.

**Reply by piroor** [reply]
And there is a workaround: TST options => "Appearance" => "Style of contents ..." => "Left side".
